modules/sso-rbac/README.md

```diff
@@ -0,0 +1,38 @@
+<!-- BEGIN_TF_DOCS -->
```

Comment:
have some minimal description of module here

Reply:
Actually there is an example directory, and in that directory there are two files describing module usage

Reply:
this is not about example, just have some description what this module is for, where how it works,
no muck words just basic description for one who open it and want to understand what it is for